release_tester/arangodb/starter/deployments/selenium_deployments/sbase.py

```diff
@@ -237,9 +238,7 @@ def detect_version(self):
                     ret['version'], ret['enterprise']))
                 if (len(ret['version']) > 0) and (len(ret['enterprise']) > 0):
                     return ret
-                else:
-                    self.progress('retry version.')
-                    continue
+                self.progress('retry version.')
```

Comment:
Was removal of continue intentional here?

Reply:
pylint pointed out that this was useless - and indeed it was.